src/js/components/Tabs/Tabs.js

```diff
@@ -53,12 +276,15 @@ const Tabs = forwardRef(
         return {
           activeIndex,
           active: activeIndex === index,
+          index,
+          tabsContextRef: tabRefs[index],
```

Comment:
I think the key/name should just be `ref`. For example, we don't say `tabsContextIndex`. If the consumer needs to rename it, it can do that as needed when using the context value.